strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout-provider/src/main/java/org/carlspring/strongbox/storage/validation/exceptions/LowercaseValidationException.java

```diff
@@ -0,0 +1,32 @@
+package org.carlspring.strongbox.storage.validation.exceptions;
```

Comment:
Similar to my comment regarding not having to have separate packages called `org.carlspring.strongbox.*.interface`, we should avoid having `org.carlspring.strongbox.storage.validation.exceptions`. In such cases, just add the respective exception, or interface class, under the base package and that should be alright.

Reply:
Done

Reply:
Did you change this?

Reply:
Yes